apel/db/unloader.py

```diff
@@ -119,8 +119,8 @@ def unload_gap(self, table_name, start, end, ur=False):
         '''
         record_type = self.RECORD_TYPES[table_name]
 
-        if record_type != JobRecord:
-            raise ApelDbException("Can only gap publish for JobRecords.")
+        if record_type != JobRecord and record_type != SummaryRecord:
```

Comment:
```suggestion
        if record_type not in (JobRecord, SummaryRecord):
```
This is the tidier way to do this.